src/Client.php

```diff
@@ -1146,8 +1147,22 @@ public function updateCronJob(string $name, string $image_name, string $schedule
   /**
    * {@inheritdoc}
    */
-  public function deleteCronJob(string $name) {
-    return $this->apiCall(__METHOD__, $name);
+  public function deleteCronJob(string $name, string $label = NULL) {
+    // If the name was passed in, just delete that specific one.
+    if (!empty($name)) {
+      return $this->apiCall(__METHOD__, $name);
+    }
+
+    // If there was no name, but is a label, delete all jobs that match.
```

Comment:
So this also deletes all jobs not just a singular one...

Reply:
Yep, when its a label, all jobs that match, gone. Pfft.